src/js/product.ts

```diff
@@ -20,15 +20,46 @@ export default () => {
 
   function onProductSlide(event: ProductSlideEvent): void {
     const thumbnails = document.querySelectorAll(SelectorsMap.product.thumbnail);
+    const thumbContainer = document.querySelector(SelectorsMap.product.thumbContainer);
```

Comment:
`thumbContainer` is single `thumbnail` in selectors-map. I believe this should be `thumbsContainer` although selectors-map still uses `thumbnailsContainer`